### DISCUSSION TITLE
Consuming .NET Feature Flags within an Angular Application

### DISCUSSION THREAD

**Discussion by giscus** [timdeschryver/timdeschryver.dev#342] [Announcements]
# Consuming .NET Feature Flags within an Angular Application

Learn how to consume feature flags from a .NET API (using Microsoft.FeatureManagement.AspNetCore) within an Angular application.

https://timdeschryver.dev/blog/consuming-net-feature-flags-within-an-angular-application

**Comment by JIBIN-P** [comment]
Hii, is there an example repo for this blog?

**Reply by timdeschryver** [reply]
Hi, sorry there isn't a example repo for this available.
